hc/api/tests/test_create_check.py

```diff
@@ -11,32 +11,41 @@ def setUp(self):
         super(CreateCheckTestCase, self).setUp()
 
     def post(self, data, expected_error=None):
-        r = self.client.post(self.URL, json.dumps(data),
-                             content_type="application/json")
+        response = self.client.post(self.URL, json.dumps(data),
+                                    content_type="application/json")
 
         if expected_error:
-            self.assertEqual(r.status_code, 400)
-            ### Assert that the expected error is the response error
+            self.assertEqual(response.status_code, 400)
+            # Assert that the expected error is the response error
+            self.assertIn(str(expected_error), str(response._container))
 
-        return r
+        return response
 
     def test_it_works(self):
-        r = self.post({
+        '''
+        checks that pings are working
+        '''
+        check = Check()
+        check.n_pings = 10
+
+        response = self.post({
             "api_key": "abc",
             "name": "Foo",
             "tags": "bar,baz",
             "timeout": 3600,
             "grace": 60
         })
 
-        self.assertEqual(r.status_code, 201)
+        self.assertEqual(re.status_code, 201)
```

Comment:
Is this running successfully? `re` or `response`?

Reply:
Worked on. Renamed the variable to response